Automattic/wp-calypso
# Stats: Adjust post detail layout for FixedNavigationHeader and Highlight cards

#### Proposed Changes

* Adjust `FixedNavigationHeader` to fit the modernized layout (`max-width: 1224px`).
* Add `min-width` to Highlight cards for scrolling on the narrower viewport.
* Modify Highlight card block styles on mobile.

#### Testing Instructions



* Spin up this change with the Calypso Live link.
* Navigate to the Stats `Post Detail` page (`/stats/post/${post-id}/${site-id}`).
* Ensure the post detail page works as previously.
* Append the feature flag `?flags=stats/enhance-post-detail` to the URL.
* Ensure the layout of the fixed navigation header aligns with other Stats pages.

| Before | After |
| --- | --- |
| <img width="1483" alt="截圖 2022-12-27 下午10 32 37" src="https://user-images.githubusercontent.com/6869813/209685157-17a02195-50b8-4523-ae8e-1c2b74340f8e.png"> | <img width="1478" alt="截圖 2022-12-27 下午10 32 57" src="https://user-images.githubusercontent.com/6869813/209685253-f4b735e5-ef63-4194-9339-b8231a845fa6.png"> |

* Ensure the Highlight cards look reasonable on narrower and mobile viewports.

| Narrower | Mobile |
| --- | --- |
| <img width="738" alt="截圖 2022-12-27 下午10 49 20" src="https://user-images.githubusercontent.com/6869813/209685390-6da62705-3be2-49cb-b51e-95db09782d68.png"> | <img width="610" alt="截圖 2022-12-27 下午10 48 37" src="https://user-images.githubusercontent.com/6869813/209685463-64dd37c1-9107-4ae9-bce6-ff79b510bcc5.png"> |

#### Pre-merge Checklist





Related to #70671
